packages/ui/src/common/components/forms/Calendar/components.tsx

```diff
@@ -0,0 +1,174 @@
+import { addDays, addMonths, isAfter, isBefore, isEqual, isSameMonth } from 'date-fns'
+import React from 'react'
+import styled, { css } from 'styled-components'
+
+import { TextMedium, TextSmall } from '@/common/components/typography'
+import { Colors } from '@/common/constants'
+import { DateSelection, PartialDateRange } from '@/common/types/Dates'
+import { fromRange } from '@/common/utils/dates'
+
+interface CalendarNavProps {
+  month: Date
+  direction: -1 | 1
+  min?: Date
+  max?: Date
+  onClick?: (month: Date) => void
+  children?: React.ReactNode
```

Comment:
This is fine but there is no need to explicit declare `children` in Functional Component. Example below is equal and in my opinion more DRY because you don't have to redeclare `children` in every component:
```
import React, { FC } from 'react'
export const CalendarNav: FC<CalendarNavProps> = ({ month, direction, min, max, children, onClick }) => {})
```

Reply:
Hi @wyrdrender,

Thanks for the review.

Just for clarifying I also have a slight preference for using `FC` but @jodator told me that [it is less consistent with rest of the project](https://github.com/Joystream/pioneer/pull/468#discussion_r621876190). That said, that other component wasn't rendering any children... So I can be clear on this point, is it consistent with the project code base to use the `FC` type only when declaring a functional component which renders children ?

Reply:
> So I can be clear on this point, is it consistent with the project code base to use the `FC` type only when declaring a functional component which renders children ?

I'm for this :+1: 

This way we have a clear definition of whether the component receives children or not.